tiny_tokenizer/word_tokenizers/mecab_tokenizer.py

```diff
@@ -43,8 +43,21 @@ def tokenize(self, text: str):
         if self.with_postag:
             for elem in parse_result.split("\n")[:-1]:
                 surface, feature = elem.split()
-                postag = feature.split(",")[0]
-                return_result.append(Token(surface=surface, postag=postag))
+                postag, postag1, postag2, postag3, \
+                    inflection, conjugation, \
+                    original_form, yomi, pron = feature.split(",")
```

Comment:
you use `origin_form` and `original_form`. I think they should be unified.

Reply:
You're right.